en/subscription_services_tos.md

```diff
@@ -0,0 +1,111 @@
+
+#### Your Privacy
+The [Mozilla Subscription Services Privacy Notice](https://www.mozilla.org/privacy/subscription-services) explains what information is sent when you use the Services and how we handle and share that information.
+
+Mozilla is a global organization, and our computers and service providers’ computers may be in various countries around the world, including the United States. This means that your information might be processed on servers located outside of the country where you live, and that country may have a different level of data protection regulation than yours. By giving us information, you consent to this kind of transfer of your information.
+
+#### Your Payment
+__Payment.__ We offer the Services as automatically renewing subscription services. When you sign up, you authorize us to charge the payment method you provide for the subscription fees for the first term of the Service(s) you sign up for. Your plan renews automatically at the end of each term, and you authorize us to charge the subscription fees each term. If you stop paying, we will immediately suspend your account, and we may delete it after 2 months of not paying.
+
+__30-Day Refund Policy.__ The first time you subscribe to the Services through Mozilla's website, if you cancel your account within the first 30 days, you may request a refund and Mozilla will refund your first subscription term. This offer only applies the first time you subscribe. This is in addition to any rights you may have under your country’s consumer laws.
+Purchasing through Apple or Google App Stores. If you purchased your subscription through in-app purchase from the Apple App Store or the Google Play Store, your payment is subject to the terms and conditions of the App Store. You must direct any billing and refund inquiries for such purchases to Apple or Google, as appropriate.
+
+#### You May Cancel Your Subscription at Any Time
+You may cancel your subscription to the Services at any time. You can do this by clicking the “Cancel Subscription” link in any email that we send you about the Services or by going to the [Subscriptions & Payments](https://accounts.firefox.com/subscriptions/) section of your Firefox Account and clicking “Cancel.” If you choose to cancel, auto-renewal will stop, we will not charge you for any future payment periods, and you will lose access to the Services at the end of the current billing cycle.
+
+## Mozilla VPN
+
+We only offer Mozilla VPN as a paid subscription service.
+
+#### You Must Be Eligible
+You must be in a country where the Service is available to download Mozilla VPN and use the Service. Currently, the Service is available in Austria, Belgium, Canada, Finland, France, Germany, Ireland, Italy, Malaysia, the Netherlands, New Zealand, Singapore, Spain, Sweden, Switzerland, the United Kingdom, and the United States. 
```

Comment:
Do we want to actually list the countries here? This is going to make this document extremely volatile, if it needs to be translated.

Reply:
for now, yes because we're limiting countries and we want to be specific until we open up more broadly.